backend/open_webui/utils/oauth.py

```
@@ -238,206 +240,336 @@ async def login(self, request, provider):
                 detail=f"Error redirecting to {provider}: {str(e)}"
             )
 
-    async def handle_callback(self, request, provider, response=None):
-        """Handle the callback from the OAuth provider"""
-        # This is a common place where errors occur, so log the request
-        log.error(f"Handling OAuth callback for provider: {provider}")
+    def _get_provider_data(self, provider_name):
```

Ruff (0.8.2)

Missing return type annotation for private function `_get_provider_data`

(ANN202)